include/accessor.h

```diff
@@ -399,6 +407,18 @@ accessor(const buffer<T, D>& buff, handler& cgh, Functor rmfn, TagT tag, propert
 template <typename DataT, int Dims, access_mode Mode>
 class accessor<DataT, Dims, Mode, target::host_task> : public detail::accessor_base<DataT, Dims, Mode, target::host_task> {
   public:
+	CELERITY_DETAIL_HACK_CLANG_TIDY_ALLOW_NON_CONST(accessor)
+	accessor(const accessor&) = default;
+	accessor& operator=(const accessor&) = default;
+	// The ComputeCpp legacy compiler seems to think that these can throw
+#if CELERITY_DETAIL_IS_OLD_COMPUTECPP_COMPILER
+	accessor(accessor&&) = default;
+	accessor& operator=(accessor&&) = default;
+#else
```

Comment:
Does this mean they can't be defaulted? Hm, probably fine as a hack for a compiler whose days are numbered.